crates/store/re_types/definitions/rerun/archetypes/ellipsoids.fbs

```diff
@@ -0,0 +1,63 @@
+include "fbs/attributes.fbs";
+include "rust/attributes.fbs";
+include "cpp/attributes.fbs";
+
+include "rerun/datatypes.fbs";
+include "rerun/components.fbs";
+
+namespace rerun.archetypes;
+
+// ---
+
+/// 3D ellipsoids with half-extents and optional center, rotations, rotations, colors etc.
```

Comment:
Let's not bury the lede: the first line is what is shown at https://rerun.io/docs/reference/types/archetypes so let's put the word "sphere" in there.

And I don't think we need to list all the ways it can be customized (I know we do at other places, but… I'm not sure its good there either :))

```suggestion
/// 3D ellipsoids or spheres.
```